docs/How/integration_checklist.md

```diff
@@ -0,0 +1,102 @@
+# Steps for a New Integration
+
+## Overview
+
+Below, you will find a checklist of the steps that should be followed by any project maintainers when performing a new
+upstream integration. This is for reference by the maintainers and by any of the community interested in the process.
+
+We will do our best to keep this up to date, so it's an excellent reference for what to expect if you're waiting on an
+integration.
+
+## 1) Announce
+
+Advertise the upcoming update, where it will start from, how long it will take, and what will happen with the existing
+release branch afterwards. This should be done in the `Release Announcements` channel in the Project Mu Team.
+
+## 2) Rebase and Test
+
+Much of this process is designed to be run on every Mu repo. When deciding which repos to update first, refer to the CI
+dependencies documented in each repo.
+
+There are currently exceptions to this rule. The `mu_basecore`, `mu_silicon_arm_tiano`, and `mu_tiano_plus` repos have a
+circular dependency that must be worked with care. Refer to the integration guide videos for clarification on how these
+three should work. They should be rebased and tested prior to any of the others, starting with `mu_basecore`.
+
+### a) Prep for the Naive Rebase
+
+* Create `rebase/*_naive` branch
```

Comment:
Although it might be obvious to some, I think it'd be better to explicitly state where to branch `rebase/*_naive` from